### ISSUE TITLE
Uncaught TypeError: Cannot read property 'navigateTo' of undefined

### ISSUE THREAD

**Issue by nosami** [OmniSharp/omnisharp-atom#250]
[Enter steps to reproduce below:]

1. ... Click on a usage in the Find panel when a cs file isn't in focus.

**Atom Version**: 0.198.0
**System**: Mac OS X 10.10.3
**Thrown From**: [omnisharp-atom](https://github.com/OmniSharp/omnisharp-atom) package, v0.4.22

### Stack Trace

Uncaught TypeError: Cannot read property 'navigateTo' of undefined

```
At /Users/jason/.atom/packages/omnisharp-atom/lib/omni-sharp-server/omni.js:34

TypeError: Cannot read property 'navigateTo' of undefined
  at Function.Omni.navigateTo (/Users/jason/.atom/packages/omnisharp-atom/lib/omni-sharp-server/omni.js:34:50)
  at ViewModel.Vue.methods.gotoUsage (/Users/jason/.atom/packages/omnisharp-atom/lib/omnisharp-atom/views/find-pane-view.js:81:26)
  at HTMLTableRowElement.handler (/Users/jason/.atom/packages/omnisharp-atom/node_modules/vue/src/directives/on.js:34:31)

```

### Commands

```
  6x -1:30.4.0 vim-mode:move-to-next-word (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -1:27.9.0 omnisharp-atom:go-to-definition (atom-text-editor.editor.vim-mode.command-mode.is-focused)
 37x -1:26.5.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.command-mode.is-focused)
 51x -1:24.7.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.command-mode.is-focused)
 27x -1:17.7.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.command-mode.is-focused)
  2x -1:16 vim-mode:move-up (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -1:14.1.0 vim-mode:find-backwards (atom-text-editor.editor.vim-mode.command-mode.is-focused)
  3x -1:14 blur (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -1:11.6.0 vim-mode:repeat (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -1:10.6.0 omnisharp-atom:go-to-definition (atom-text-editor.editor.vim-mode.command-mode.is-focused)
  2x -1:10.6.0 blur (atom-text-editor.editor.vim-mode.command-mode.is-focused)
 50x -1:09.7.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.command-mode.is-focused)
 46x -1:07.1.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.command-mode.is-focused)
  5x -0:54.4.0 blur (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -0:48 tree-view:toggle (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
 26x -0:48 blur (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "metrics",
      "autocomplete",
      "web-browser"
    ],
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "themes": [
      "atom-dark-ui",
      "gruvbox"
    ]
  }
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
atom-browser-webview, v0.6.0
atom-typescript, v4.1.5
atom-yeoman, v0.2.0
autocomplete-plus, v2.12.1
gruvbox, v0.3.2
language-csharp, v0.5.0
linter, v0.12.1
omnisharp-atom, v0.4.22
vim-mode, v0.46.0

# Dev
No dev packages
```

**Comment by david-driscoll** [comment]
Try `0.4.33` and let us know if it's still an issue!

**Comment by nosami** [comment]
If it wasn't fixed in 0.4.3.33, it's definitely fixed in 0.5.0 :)
